server/paymentProviders/opencollective/host.js

```diff
@@ -16,6 +16,20 @@ paymentMethodProvider.getBalance = () => {
   return Promise.resolve(maxInteger);
 };
 
+paymentMethodProvider.createPlatformTipTransaction = async transaction => {
+  const platformTipTransaction = {
+    ...transaction,
+    amount: transaction.platformTipInHostCurrency,
+    description: 'Financial contribution (Platform Tip) to Open Collective',
+    netAmountInCollectiveCurrency: transaction.platformTipInHostCurrency,
+    FromCollectiveId: transaction.HostCollectiveId,
+    CollectiveId: 8686, // Open Collective (Organization)
+    platformTipInHostCurrency: undefined,
```

Comment:
Was this checked with @kewitz. Will this be taken in consideration in the settlement invoice?

Reply:
If this is similar to a Bank Transfer then it should be good.

Reply:
@kewitz : I think you reviewed this before; but if you don't mind can you do a review again as we changed the code a bit (mostly on the front-end) with the new design updates. 😄

Reply:
@SudharakaP
This is missing `PlatformTipForTransactionGroup` and the value should be calculated and stored in USD, OpenCollective's currency.
`data.hostToPlatformFxRate` will also need to be calculated in the case host's currency is not USD.
I strongly recommend using `Transaction.createFeesOnTopTransaction` through `Transaction.createFromPayload`.
Check `createFeesOnTopTransaction` out and let me know how this differs from this one and we can discuss the best way forward.